pod/completion/views.py

```diff
@@ -169,9 +170,10 @@ def video_caption_maker_save(request, video):
 
 @csrf_protect
 @login_required(redirect_field_name="referrer")
-def video_completion(request, slug):
+def video_completion(request: WSGIRequest, slug: str):
     """Video Completion view."""
     video = get_object_or_404(Video, slug=slug, sites=get_current_site(request))
+    page_title = _("Video additions of the video “%s”") % video.title
```

Comment:
Ptet juste `Additions for the video “%s”` pour pas répéter 2X video ?

Reply:
Tout a fait !

Reply:
Puis je vais en faire une variable globale au fichier. Pour éviter de réécrire ce `page_title` à chaque fois !